src/controllers/abstract-controller.ts

```diff
@@ -39,5 +51,21 @@ export default abstract class AbstractController {
     }
   }
 
+  // Argument 'data' it is a new data, and argument existedData is optional and needed for refreshing existing data in db
+  protected async validate(data: any, existedData: any = {}) {
+    const createdModel = await this.entity.create(Object.assign(existedData, data));
+    const errors = await validate(createdModel);
+    if (errors.length) {
+      const parsedErrors = errors.reduce(
+        (acc, error) => ({
```

Comment:
please, add typings to reduce callback params

Reply:
Done.